code/src/app/unite/site/site.service.ts

```diff
@@ -77,4 +77,9 @@ export class WidgetService {
                         return checkArr;
                     })
     }
+
+    getConfigs()
```

Comment:
@ttplsudhir  - Rename this method to `getGlobalConfigs`

Reply:
@sushan05 done